.devcontainer/docker-compose.yml

```
@@ -0,0 +1,67 @@
+    container_name: cache_database_dozens
+    restart: unless-stopped
+    networks:
+      - dozens-ecosystem
+    volumes:
+      - /home/terry-brooks/Github/the-dozens-django/redis.conf:/usr/local/etc/redis/redis.conf:ro
+      - /home/terry-brooks/Github/the-dozens-django/users.acl:/usr/local/etc/redis/users.acl:ro
+
+    ports:
+      - '3001:6379'
+    healthcheck:
+      test: [ "CMD", "redis-cli", "--raw", "incr", "ping" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)